primer/primitives
# Adds tokens for inactive button state

## Summary

Adds color tokens to support Inactive button state styles

<img width="589" alt="Screenshot 2023-11-09 at 2 38 52 PM" src="https://github.com/primer/primitives/assets/2313998/9eb1f904-2d91-4599-90cd-4977daf63eae">


(From [this Figma](https://www.figma.com/file/0azGmqesYBX2iEsbbR4ikH/%5BExplorations%5D---Graceful-degradation-patterns?type=design&node-id=120-948&mode=design))


## List of notable changes:

- **added** inactive button text colors:
  - New build: `button-inactive-fgColor-rest`
  - Legacy build: `btn.inactive.text`
- **added** inactive button text colors:
  - New build: `button-inactive-bgColor-rest`
  - Legacy build: `btn.inactive.bg`



## What should reviewers focus on?

- Did I add these correctly?

## Steps to test:

1. Upgrade your app to use the Canary build version
2. Try using these tokens to style inactive buttons
3. Confirm the correct colors are rendered correctly in all color modes

You can use this [@primer/react PR](https://github.com/primer/react/pull/3812) to test.

## Supporting resources (related issues, external links, etc):

- [Figma exploration](https://www.figma.com/file/0azGmqesYBX2iEsbbR4ikH/%5BExplorations%5D---Graceful-degradation-patterns?type=design&node-id=120-948&mode=design)
- https://github.com/primer/react/pull/3812

## Contributor checklist:


## Reviewer checklist:
